.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -5,38 +5,109 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
-        Please make sure to fill out the entire form below,
-        providing as much context as you can in order to help us
-        triage and track down your bug as quickly as possible.
+        > [!IMPORTANT]
+        > **Please read all parts of this form carefully.** By following the instructions carefully, you ensure that we
+        > can get started fixing your bug, instead of being stuck at trying to reproduce your issue. Please provide
+        > all requested information, even if you think it does not apply to your problem.
 
-        Before filing a bug, please be sure you have searched through
-        [existing bugs](https://github.com/open-telemetry/opentelemetry-js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug)
-        to see if your bug is already addressed.
-
-        If your bug is related to an instrumentation or plugin in [opentelemetry-js-contrib](https://github.com/open-telemetry/opentelemetry-js-contrib)
-        please be sure to file it there.
+        Before filing a bug, please be sure you have searched through [existing open bugs](https://github.com/open-telemetry/opentelemetry-js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug)
+        to see if this bug has already been filed.
+  - type: markdown
+    attributes:
+      value: |
+        ## Basic Troubleshooting
+  - type: checkboxes
+    id: updated-to-latest
+    attributes:
+      description: |
+        > [!NOTE]
+        > If you depend on experimental packages with a range like `opentelemetry/sdk-node@^0.53.0` that is equivalent
+        > to `@opentelemetry/sdk-node@0.53.x`, NOT `@opentelemetry/sdk-node@0.x.x`. `0.54.x` **WILL NOT** be picked up
+        > by this constraint due to the way semantic versioning works for `0.x` packages. This is intentional, see
+        > https://semver.org/ for details.
+      label: Updating packages
+      options:
+        - label: I have updated all `@opentelemetry/*` packages but the issue persists
+          required: true
```

Comment:
I hesitate slightly at `required: true` here. If I want to report an issue specifically with an older version then there is no way to move on from this. (This will be more relevant when we have 2.x current versions *and still supported 1.x versions for a period of time.)

Granted that gives the more lazy bug reporter an "out". I'm conflicted. :)

Reply:
I think we'd have to update the bug report template when we release 2.0 - for now this still holds and I believe it would free up a lot of our time. Right now a lot of issues are solved by updating, and we don't support older versions unless they are critical vulnerabilities, even there we've only backported it for 2 versions before (and only once as far as I can remember). I feel like we're also not in a position to change our support to anything other than latest (and last major) anytime soon.

I'd follow up with adding an issue that we update this statement in preparation for the SDK 2.0 release, WDYT?

Reply:
Sounds good to tweak this after 2.0.